doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -5145,82 +5151,119 @@ \subsubsection{Inputs}\label{wwhpeir_inputs}
 
-\paragraph{Water to Water Heat Pump Load Side Heat Transfer Energy {[}J{]}}\label{eir_wwhp_outputs_q_load_sum}
+\paragraph{Plant Loop EIR Heat Pump Load Side Heat Transfer Energy {[}J{]}}\label{plhp_eir_outputs_q_load_sum}
 
 This output variable represents the summed load side heat transfer during the specified output reporting window.
 
-\paragraph{Water to Water Heat Pump Source Side Heat Transfer Rate {[}W{]}}\label{eir_wwhp_outputs_q_source}
+\paragraph{Plant Loop EIR Heat Pump Source Side Heat Transfer Rate {[}W{]}}\label{plhp_eir_outputs_q_source}
 
 This output variable represents the average source side heat transfer occurring for this coil during the specified output reporting window.
 
-\paragraph{Water to Water Heat Pump Source Side Heat Transfer Energy {[}J{]}}\label{eir_wwhp_outputs_q_src_sum}
+\paragraph{Plant Loop EIR Heat Pump Source Side Heat Transfer Energy {[}J{]}}\label{plhp_eir_outputs_q_src_sum}
 
 This output variable represents the summed source side heat transfer during the specified output reporting window.
 
-\paragraph{Water to Water Heat Pump Load Side Inlet Temperature {[}°C{]}}\label{eir_wwhp_outputs_t_load_in}
+\paragraph{Plant Loop EIR Heat Pump Load Side Inlet Temperature {[}°C{]}}\label{plhp_eir_outputs_t_load_in}
 
 This output variable represents the inlet temperature on the load side connection of this coil.
 
-\paragraph{Water to Water Heat Pump Load Side Outlet Temperature {[}°C{]}}\label{eir_wwhp_outputs_t_load_out}
+\paragraph{Plant Loop EIR Heat Pump Load Side Outlet Temperature {[}°C{]}}\label{plhp_eir_outputs_t_load_out}
 
 This output variable represents the outlet temperature on the load side connection of this coil.
 
-\paragraph{Water to Water Heat Pump Source Side Inlet Temperature {[}°C{]}}\label{eir_wwhp_outputs_t_source_in}
+\paragraph{Plant Loop EIR Heat Pump Source Side Inlet Temperature {[}°C{]}}\label{plhp_eir_outputs_t_source_in}
 
 This output variable represents the inlet temperature on the source side connection of this coil.
 
-\paragraph{Water to Water Heat Pump Source Side Outlet Temperature {[}°C{]}}\label{eir_wwhp_outputs_t_source_out}
+\paragraph{Plant Loop EIR Heat Pump Source Side Outlet Temperature {[}°C{]}}\label{plhp_eir_outputs_t_source_out}
 
 This output variable represents the outlet temperature on the source side connection of this coil.
 
-\paragraph{Water to Water Heat Pump Load Side Mass Flow Rate {[}kg/s{]}}\label{eir_wwhp_outputs_load_flow}
+\paragraph{Plant Loop EIR Heat Pump Load Side Mass Flow Rate {[}kg/s{]}}\label{plhp_eir_outputs_load_flow}
 
 This output variable represents the average load side mass flow rate during the specified reporting window.
 
-\paragraph{Water to Water Heat Pump Source Side Mass Flow Rate {[}kg/s{]}}\label{eir_wwhp_outputs_src_flow}
+\paragraph{Plant Loop EIR Heat Pump Source Side Mass Flow Rate {[}kg/s{]}}\label{plhp_eir_outputs_src_flow}
```

Comment:
This report variable change will trigger an addition to the Report Variables transition support file.